lib/src/main/java/com/deblock/cucumber/datatable/annotations/Ignore.java

```diff
@@ -0,0 +1,13 @@
+package com.deblock.cucumber.datatable.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * This annotation signals a class/field that should not be mapped
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.TYPE, ElementType.FIELD, ElementType.RECORD_COMPONENT, ElementType.PARAMETER})
```

Comment:
why use `@Ignore` on TYPE?

Reply:
I just see that this annotation can be use to ignore class registration. 
With the new registration behavior, the library will register all classes, so the annotation  `DataTableWithHeader` become useless.
I don't see why @Ignore will be used. 

You say that 
```
@DataTableWithHeader // this annotation is used to register this class as a Datatable
                     // This is facultative but may improve test startup performance
```
But you will always check all classes, the startup performance will not be really improved. 

I think that we should be able to provide a configuration entrypoint to be able to choose the loading process: 
- register only `@DataTableWithHeader` classes
- or check classes with public setter/public fields (so don't use `@DataTableWithHeader` annotation)